### ISSUE TITLE
Su-30SM Flanker G - 2019

### ISSUE THREAD

**Issue by tbrandon517** [PygmalionOfCyprus/cmo-db-requests#470]
### DB Selector

DB3K

### DBID to Copy

aircraft/4959/DB3K

### Hypothetical

No

### Name

Su-30SM Flanker G - 2019

### Country

Armenia

### Service

Air Force

### In Commission

2019-

### Sensors

None Known

### Mounts

None Known

### Loadout Options

Remove AAM loadouts for all except those with R-77M and/or R-27R

Remove all guided air-to-surface ordnance loadouts, keep only unguided munitions.

### Other Changes

None Known

### Sources

https://www.oryxspioenkop.com/2022/01/knights-of-yerevan-armenias-su-30sm.html?m=1&fbclid=IwAR2MV6ynh8bniYCajh0FX7BU6OY7CUc35yORdnx4KjSafiezImRPbuFGJKk
https://en.wikipedia.org/wiki/Sukhoi_Su-30#Operators
https://www.flightglobal.com/analysis/analysis-2019-world-air-forces-directory/130525.article

**Comment by PygmalionOfCyprus** [comment]
I assume you meant R-73M and not R-77M.

Anyway, added 494. I removed all its loadouts except for unguided weapons as requested; I also removed its A/A loadouts short of the plain light/heavy R-27/R-73 mix. It's not clear whether they received any R-27ERs, R-27Ts, etc. so until we get indication otherwise I think it's probably better just to keep it only to R-27Rs.

That said, I haven't deleted the alt A/A loadouts, just removed them from the AC; they can be added with Lua or easily in the future if we get more info and it turns out they did receive a wider missile package.
